sql/plan/show_create_table.go

```diff
@@ -191,33 +193,42 @@ func (i *showCreateTablesIter) Next(ctx *sql.Context) (sql.Row, error) {
 
 	i.didIteration = true
 
-	var composedCreateTableStatement string
-	var tableName string
-
+	var row sql.Row
 	switch table := i.table.(type) {
 	case *ResolvedTable:
 		// MySQL behavior is to allow show create table for views, but not show create view for tables.
 		if i.isView {
 			return nil, ErrNotView.New(table.Name())
 		}
 
-		tableName = table.Name()
-		var err error
-		composedCreateTableStatement, err = i.produceCreateTableStatement(ctx, table.Table, i.schema, i.pkSchema)
+		composedCreateTableStatement, err := i.produceCreateTableStatement(ctx, table.Table, i.schema, i.pkSchema)
 		if err != nil {
 			return nil, err
 		}
+		row = sql.NewRow(
+			table.Name(),                 // "Table" string
+			composedCreateTableStatement, // "Create Table" string
+		)
 	case *SubqueryAlias:
-		tableName = table.Name()
-		composedCreateTableStatement = produceCreateViewStatement(table)
+		characterSetClient, err := ctx.GetSessionVariable(ctx, "character_set_client")
```

Comment:
This should be the charset/collation from the session when the view was created, but I see that we're doing this the same way in other code, like `show_create_procedure.go`, so this seems ok for now until we tackle better charset support.

Reply:
Yep, we currently support only one character set/collation.